src/Sql/dbo/Stored Procedures/User_Create.sql

```diff
@@ -14,6 +14,7 @@
     @AccountRevisionDate DATETIME2(7),
     @Key NVARCHAR(MAX),
     @PublicKey NVARCHAR(MAX),
+    @SignedPublicKeyOwnershipClaim NVARCHAR(MAX),
```

Comment:
🎨 New columns or parameters should always be added at the end and for a proc you want them to be nullable e.g. `= NULL`.